ztp/argocd/deployment/argocd-controller-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
we already have it, under https://github.com/serngawy/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/gitops-cluster-rolebinding.yaml

Reply:
fixed